Backend/app/dashboard/Lib/src/Tools/Client.hs

```diff
@@ -84,6 +85,7 @@ instance
 type ApiWithMerchant api =
   "dashboard"
     :> Capture "merchantId" (CheckedShortId DM.Merchant)
+    :> Capture "city" City.City
```

Comment:
backward incompatible.

Reply:
I have created a new client, but it will backward compatible as old API and APIv2 will send city and this client is only used in dashboard. It should work right?